src/vmm/src/vstate/vcpu/mod.rs

```diff
@@ -450,13 +450,15 @@ impl Vcpu {
             Ok(run) => match run {
                 VcpuExit::MmioRead(addr, data) => {
                     if let Some(mmio_bus) = &self.kvm_vcpu.mmio_bus {
+                        let _metric = METRICS.vcpu.exit_mmio_read_agg.record_latency_metrics();
```

Comment:
Why only x86?

Reply:
because we don't have these cases for aarch64 https://github.com/sudanl0/firecracker/blob/main/src/vmm/src/vstate/vcpu/aarch64.rs#L231

Reply:
Oh my bad, I thought this file was x86 specific. Ignore me!